### PR TITLE
Add in node defaults

### PR DESCRIPTION
**Pull request by nolim1t**
This is still a preview

## Still a lot of stuff Todo but we have a base system:

- [x] Add in docker-compose file (to be changed later). We would like to get nginx added in here if possible.
- [x] Add docker-compose init.d and also link to runlevels (i.e. /etc/rc2.d)
- [x] Configure services linked to docker-compose
- [x] Test out docker-compose to see it all start up.
- [x] Define real services (e.g. nginx, bitcoind?, lnd?, invoicer?, Another system service for handling createwallet and node unlocks?). This should be done in this issue #13 (so we keep this less cluttered) 
- [x] Configure real services. Same as above.

## Temporary stuff while developing

- [x] Create wallet and unlock scripts (Re-use the existing ones), so we can can get a system up and running asap. This should be superceded by a system service.
- [x] Use neutrino because #4 isn't yet done (and for developing without waiting to sync a full bitcoind, but maybe it can be a permanent thing. We configure neutrino while waiting to sync and then switch over.

## Other Ideas

- [ ] Tor container (with permissions / cookies), so we don't need the system tor. Or if LND starts supporting passwords, we can use this instead.

### CODE REVIEW

**Comment by nolim1t** [comment]
docker-compose will be called compose-service in init.d scripts

**Comment by nolim1t** [comment]
Currently testing out the latest service changes

**Comment by nolim1t** [comment]
symlinking might be an issue.. maybe put it inside the chroot

**Comment by nolim1t** [comment]
## Status update

We now have a service called ```umbrelbox``` which is in the system runlevels starts up docker-compose. 

docker-compose.yml is in ```$HOME``` (pathname is /home/umbrel)

**Comment by nolim1t** [comment]
Another point to consider.

Should we run createwallet in rc.local or within a container?

I'm leaning more towards a container so its easier to test/debug.

File: stage2/07-configure-LND/files/lnd.conf [hunk 1]
```diff
@@ -4,9 +4,9 @@ listen=0.0.0.0:9735
 rpclisten=0.0.0.0:10009
 restlisten=0.0.0.0:8080
 maxpendingchannels=3
-minchansize=10000
+minchansize=1337
 alias=Umbrel Node
-color=#6320fc
+color=#5351FB
```
Comment (nolim1t) [inline stage2/07-configure-LND/files/lnd.conf]: @mayankchhabra  :D

**Comment by nolim1t** [comment]
This should be almost ready for a new release soon!

**Comment by nolim1t** [comment]
pre-release image available [here](https://github.com/getumbrel/os-base/releases/tag/0.0.1.8).. feel free to test

**Comment by mayankchhabra** [comment]
Yo @nolim1t, revisiting your last idea regarding Tor. Now that LND supports passwords, should we use that or still consider dockerized Tor? Although dockerized would be more consistent with our overall architecture

**Pull request merged**